web/components/feed/feed-comments.tsx

```diff
@@ -238,7 +286,7 @@ export function getMostRecentCommentableBet(
         )
       ) {
         if (!answerOutcome) return true
-        // If we're in free response, don't allow commenting on ante bet
+        if (sortedBetsByCurrentUser[0].id !== bet.id) return false
```

Comment:
This is a little confusing. Might be clearer to add an if case at the top level of this function for `if (answerOutcome) { }`, and then consider just the first bet in `sortedBetsByCurrentUser`.

Reply:
Good idea!